### DISCUSSION TITLE
#8. VATZ biweekly meeting at 2022-05-10 22:00:00(KST) | 2022-05-11 08:00:00(CST)

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#103] [General]
# Announcement

- I'm frustrated that people do not create an issue, with required fields
![image](https://user-images.githubusercontent.com/6308023/167744131-39943f17-cbd6-41e1-b6b4-36a615f811dd.png)

- Please, create an issue that connects with PR if there's no issue on the VATZ, 
  - Do not connect PR to an issue that has label `good first issue`  
  - Create a new issue if you need one to connect PR

---

# New Updated
- Meeting time Update
  Bi-weekly (Wednesday 22:00:00 KST |  Wednesday 08:00:00 CST)

# Discussion 
- https://github.com/dsrvlabs/vatz/issues/100

**Comment by xellos00** [comment]
# 동요

- Development
   - https://github.com/dsrvlabs/vatz/issues/95
   - https://github.com/dsrvlabs/vatz/issues/98
     - https://github.com/dsrvlabs/vatz-plugin-near

- Documentation
   - https://github.com/dsrvlabs/vatz/issues/89
   - https://github.com/dsrvlabs/vatz/issues/94 
   
- Discussion
   - https://github.com/dsrvlabs/vatz/issues/75
   - https://github.com/dsrvlabs/vatz/issues/101
   - https://github.com/dsrvlabs/vatz/issues/113

**Comment by rootwarp** [comment]
# 준교

- #110 
- https://github.com/dsrvlabs/vatz/issues/106
- https://github.com/dsrvlabs/vatz/issues/107
- https://github.com/dsrvlabs/vatz/issues/108

**Comment by gnongs** [comment]
## 진홍

- https://github.com/dsrvlabs/vatz/issues/79
- https://github.com/dsrvlabs/vatz/issues/80
- https://github.com/dsrvlabs/vatz/issues/104

**Comment by kim201212** [comment]
# 형기

- #87

**Reply by xellos00** [reply]
Can you create an issue Feature request for secret?

**Reply by xellos00** [reply]
for #85 
- Create a new issue for how to run
- Code Unittest

**Comment by hqueue** [comment]
# 형규

- #72

**Comment by xellos00** [comment]
It's been a week since last Vatz bi-weekly meeting. 

@heejin-github @meetrick 
Can you guys create an issue to get started with develop a plugin for one of your protocol?
and register as child to 
- #88 

@kim201212 
Can you create the issues for followings?
- unit test documentation 
- scret_key voting for github action
